code/web/src/modules/user/Profile.js

```diff
@@ -31,16 +35,42 @@ const Profile = (props) => (
     </Grid>
 
     <Grid>
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        {/*<Tile style={{}} image={icon}></Tile>*/}
+        <ImageTile width={300} height={530} shadow={level1} image={`${ APP_URL }/images/stock/men/1.jpg`} />
```

Comment:
I am curious the sad path handling we will need once a user can upload a photo so we can rangle the dimensions into something we like more. 
I do like using a current stock photo as a work around for having something as aplaceholder at this there now.

Reply:
We may want to investigate having a set style in CSS for images uploaded so we don't run into issues with aspect ratios.